docs/src/atomistic/outputs.rst

```diff
@@ -167,3 +167,43 @@ Energy ensemble gradients
 
 The gradient metadata for energy ensemble is the same as for the ``energy``
 output (see `Energy gradients`_).
+
+.. _features:
+
+Features
+^^^^^^^^
+
+Last layer features represent the outputs generated by the model before the final predictions,
+such as energy or forces, are made. They are associated with the ``"features"`` key in the model
+outputs, and must adhere to the following metadata:
```

Comment:
These are not necessarily "last layer" features, since there might not even be multiple layer, or another prediction made by the model.

Reply:
technically these are internal/hidden/learned features. all in all, `features` might be the least bad possible name if we want to have a generic way to refer to the intermediate values in the model. maybe 'learned_features' or 'model_features' but I'm not sure there's much gain from all those extra letters.

Reply:
this output is not only intended for internal/learned features, but any kind of representation (so PLUMED would move to use this output instead of the current custom output used over there for CV)

Reply:
IMO then `features` is the most generic catch-all term - but I support polling MLDEV.